utilities/blob_db/blob_db_test.cc

```diff
@@ -104,162 +128,60 @@ class BlobDBTest : public testing::Test {
 
 TEST_F(BlobDBTest, DeleteComplex) {
   BlobDBOptionsImpl bdboptions;
-  bdboptions.partial_expiration_pct = 75;
-  bdboptions.gc_check_period_millisecs = 20 * 1000;
-  bdboptions.blob_file_size = 219 * 1024;
-
-  Reopen1(bdboptions);
-
-  WriteOptions wo;
-  ReadOptions ro;
-  std::string value;
-
-  ColumnFamilyHandle *dcfh = blobdb_->DefaultColumnFamily();
+  Reopen(bdboptions);
 
   Random rnd(301);
```

Comment:
Just an arbitrary seed. I'm hesitate to define a constant for it, because it is arbitrary and different test can use different seed.